samcli/commands/pipeline/bootstrap/guided_context.py

```diff
@@ -38,9 +39,11 @@ def run(self) -> None:
         for the pipeline to work. Users can provide all, none or some resources' ARNs and leave the remaining empty
         and it will be created by the bootstrap command
         """
+        account_id = bootstrap.get_current_account_id()
         if not self.environment_name:
             self.environment_name = click.prompt(
-                "Environment name (a descriptive name for the environment which will be deployed to this AWS account)",
+                f"""Environment name (a descriptive name for the environment which
+                 will be deployed to AWS account {account_id})""",
```

Comment:
The indentation will be printed as well. (after "which" and before "will")

Reply:
when `account_id` is `None`, it will print `AWS account None`, I think we should raise an exception or print an error message before here.

Reply:
fixed.